src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1613,7 +1613,11 @@ public function insert_posts( $items = array() ) {
 						if ( ! $venue_id ) {
 							$event['Venue']['ShowMap']     = $show_map_setting;
 							$event['Venue']['ShowMapLink'] = $show_map_setting;
-							$venue_id = $event['EventVenueID'] = Tribe__Events__Venue::instance()->create( $event['Venue'], $event['post_status'] );
+
+							$venue_id = $event['EventVenueID'] = Tribe__Events__Venue::instance()->create(
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 06632d555efb8adad62446f8cf2eab2f</sub>